src/pages/turingcup.js

```diff
@@ -1,14 +1,9 @@
 import React from "react"
-
+import Roadmap from "../components/roadmap"
 export default function App() {
   return (
-    <div className="iframe-container">
-      <iframe
-        className="iframe"
-        title="Turing Cup"
-        src="/turing_cup.pdf"
-        frameborder="0"
-      ></iframe>
+    <div>

```

Comment:
Add the Navbar component here and change the target for the anchor tag for Turing Cup in NavLink.js.  Except this everything looks fine